src/e2e/java/teammates/e2e/cases/InstructorFeedbackEditPageE2ETest.java

```diff
@@ -80,7 +80,7 @@ public void testAll() {
         questionToCopy.setQuestionNumber(2);
         feedbackEditPage.copyQuestion(copiedCourse.getId(), questionToCopy.getQuestionDetailsCopy().getQuestionText());
 
-        feedbackEditPage.verifyStatusMessage("The question has been added to this feedback session.");
+        feedbackEditPage.verifyStatusMessage("The selected question(s) have been added to this feedback session.");
```

Comment:
We added more features, so its a good idea to add more e2e tests here

Reply:
Not quite sure of what else to add for E2E tests since the overall copy question flow is generally the same, so I have left them as is. Let me know if you have any suggestions on what I can add and I'll work on it!

Reply:
@zhaojj2209  so we added some new UI components e.g the reload button, so we can add e2e tests for those

Reply:
Test added!